### ISSUE TITLE
Error when submiting worklogs if login expires

### ISSUE THREAD

**Issue by Siegrift** [vacuumlabs/fine-tracker#233]
I wanted to submit a worklog, but spinning wheel was just... spinning :D 

![sad_story](https://user-images.githubusercontent.com/22679154/68496063-c4779800-0251-11ea-8aba-6fd74f0d9fbc.png)

The error message in dev tools also doesn't suggest anything about login expiration, but after refresh and login the worklogs persisted and I could submit them.

It would be nice to check if the user is signed in before actually submitting. Another thing (that I don't know if can be improved) is to login to fine tracker only once (at least per week or so)

**Comment by mikx98** [comment]
the fact that it happens often is a duplicate of #100 and will be resolved in #207
the other feature request, the actual better error message for user not signed in is feasible - wdyt? @jakuub

**Comment by jakuub** [comment]
This should not be an issue anymore. We didn't fix the error message, but we fixed session expiration - it should not expire anymore. 

In the case of regression, let's open this again
